src/store/getters/state-getters.ts

```diff
@@ -218,6 +218,11 @@ export const getBusinessContact = (state: StateIF): BusinessContactIF => {
   return state.stateModel.defineCompanyStep.businessContact
 }
 
+export const getSnapshotContact = (state: StateIF): BusinessContactIF => {
+  const snapShotContact: unknown = state.stateModel.originalSnapshot[5]
+  return snapShotContact as BusinessContactIF
```

Comment:
👍

Reply:
Are you going to fix the other snapshot[] instances with getters?
Or next ticket?

Reply:
I think as we go. 
When the other components in Alterations are needing the specific original values for a diff, that might be a good time to add them.

Reply:
ok